models/completion_aggregator/aggregated_completion_events.sql

```diff
@@ -0,0 +1,30 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_XAPI_DATABASE", "xapi"),
+        engine=aspects.get_engine("ReplacingMergeTree()"),
```

Comment:
@bmtcril I also had to qualify this Aspects `get_engine` macro with the `aspects.` prefix.

Reply:
Oh nice, good find 👍